packages/tools/tests/test/visualization/config.json

```diff
@@ -1845,7 +1845,7 @@
         },
         {
             "title": "MeshDebugPluginMaterial",
-            "playgroundId": "#TFDNZI#2",
+            "playgroundId": "#TFDNZI#6",

```

Comment:
Shouldn't we exclude this test for WebGPU instead of forcing GLSL? I think that once we've finally ported everything to WebGPU, all the tests should work in the WebGPU engine. It'll be hard to remember that we forced GLSL for some tests, so they'll stay that way even when we run the visualization tests in WebGPU mode. On the contrary, it's easy to search for "WebGPU" in config.json to locate any tests that are excluded in WebGPU and fix them.

Reply:
Yes that makes sense!